### PR TITLE
PPSE-1454: added return after callback to avoid code execution if there is an error or if it is finished

### PR DESCRIPTION
**Pull request by paoamaralwz**
added return after callback to avoid code execution if there is an error or if it is finished

### CODE REVIEW

**Review by calexicoz — APPROVED** [review]
@paoamaralwz can you update the version number in `package.json` to `1.10.7.1`? This will be needed for the release. We should also include some notes in the `RELEASE_NOTES` file, which has not been updated properly.

Thanks!

**Comment by paoamaralwz** [comment]
> @paoamaralwz can you update the version number in `package.json` to `1.10.7.1`? This will be needed for the release. We should also include some notes in the `RELEASE_NOTES` file, which has not been updated properly.
> 
> Thanks!

ok, I will update the version and update RELEASE_NOTES

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # ANS Release Notes
 
+### 1.10.7.1 2021/3/30
+
+* Adds a return when it finish reading files to retrive ANS versions
```
Comment (calexicoz) [inline RELEASE_NOTES.md:5]:
```suggestion
* Prevent the callback function from being called twice when errors occur during the loading of schemas
```

**Review by calexicoz — COMMENTED** [review]
Hey @paoamaralwz,

Sorry I just noticed this but the issue appears for both `loadAllSchemas` and `loadSchema`. Can you fix it for the other one too?

Thanks.

**Review by calexicoz — APPROVED** [review]

**Pull request merged**